api/swagger.yaml

```diff
@@ -9247,6 +9255,7 @@ paths:
               AttachStderr: true
               DetachKeys: "ctrl-p,ctrl-q"
               Tty: false
+              ConsoleSize: nil
```

Comment:
I'm pretty sure that `nil` is not an alias for null in YAML, so this construct would be equivalent to the JSON `{"ConsoleSize": "nil"}`. I don't think that's your intention. Maybe omit it entirely from the `Tty: false` example and provide a separate example with `Tty: true` and `ConsoleSize` set to an array, like the ExecStartConfig example below?

Reply:
Thanks, good catch! See what Go does to some people... 😄 
Unfortunately multiple examples seems to be supported in OpenAPI 3.0 and we use 2.0 here (https://swagger.io/docs/specification/adding-examples/).
So I'll just leave the `Tty: false` and remove ConsoleSize from here. So we don't repeat the same example as ExecStartConfig.